pkgs/applications/networking/firewalk/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, stdenv
+, fetchurl
+, libnet
+, libpcap
+, libdnet
+}:
+
+stdenv.mkDerivation rec {
+    pname = "firewalk";
+    version = "5.0-5";
```

```suggestion
    version = "5.0";
```
The number at the end is the debian revision number, so we don't need it.